README.md

```diff
@@ -94,19 +90,18 @@ I feel like learning about Wayland is a better use of my effort.
 Lack of EWMH means some applications (especially games) will break when 
 trying to go fullscreen.
 
-Authors
-=======
-
-See [the credits](CREDITS.md) for a full list of everybody who's helped.
-
-9wm is maintained by Jacob Adams <tookmund@gmail.com>.
 
-9wm was maintained by Neale Pickett <neale@woozle.org>.
+## Authors
 
-9wm was written by David Hogan (dhog@cs.su.oz.au), a postgraduate
-student at the Basser Department of Computer Science, University
-of Sydney <http://unauthorised.org/dhog/index.html>
-David Hogan died in 2003.
+See the [CREDITS.md](CREDITS.md) for a full list of everybody who's helped.
 
+* 9wm is maintained by Jacob Adams <tookmund@gmail.com>
+* 9wm was maintained by Neale Pickett <neale@woozle.org>
 
+9wm was originally written by David Hogan <dhog@cs.su.oz.au>, a postgraduate student at the [Basser Department of Computer Science, University of Sydney](http://unauthorised.org/dhog/index.html). He passed away in 2003.
 
+[lightdm]: https://wiki.archlinux.org/title/LightDM
+[rio]: https://en.wikipedia.org/wiki/Rio_(windowing_system)
+[AUR]: https://aur.archlinux.org/packages/9wm
+[make]: https://www.gnu.org/software/make
```

Comment:
FSF did not create make, and we do not require GNU make for this package. I'm not sure what a good link would be to make, probably a 30-year-old book! But since we're not linking to other build dependencies (like a C compiler or libXext), let's not bother linking where to read more about make, either.